### PR TITLE
Front end mock

### PR DESCRIPTION
**Pull request by Behold3th**
Front end mock built separately with Next.js and then merged in. 

Current functionality includes: 
`Create`, `Remove`, `Update`, and `Describe` functionality. 

WIP: card structure to be changed to better conform with whimsical.

### CODE REVIEW

**Review by amaredeus — CHANGES_REQUESTED** [review]
There is netlify integration as seen in previous PR https://github.com/BanklessDAO/bounty-board/pull/17#issuecomment-877320039 but I'm not seeing it here

@PaulApivat how was netlify integrated?

**Comment by amaredeus** [comment]
Netlify is now integrated as part of #23

File: components/Form.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
+import { useState } from 'react'
```
Comment (amaredeus) [inline components/Form.js:1]: This file should be moved under packages/react-app/src

File: css/form.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+form {
```
Comment (amaredeus) [inline css/form.css:1]: This file should be moved under packages/react-app/src

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,184 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
```
Comment (amaredeus) [inline css/style.css:1]: Going to paste the directly the files should be moved under packages/react-app/src

File: models/Bounty.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import mongoose from 'mongoose'
+
+/* BountyBoardSchema will correspond to a collection in your MongoDB database. */
+const BountyBoardSchema = new mongoose.Schema({
```
Comment (amaredeus) [inline models/Bounty.js:4]: Going to paste the directly the files should be moved under packages/react-app/src

File: models/Bounty.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import mongoose from 'mongoose'
```
Comment (amaredeus) [inline models/Bounty.js:1]: Can we use mongodb instead of mongoose?

File: package.json [hunk 6]
```diff
@@ -1,28 +1,17 @@
 {
-  "name": "@bounty-board/monorepo",
-  "version": "1.0.0",
-  "private": true,
```
Comment (amaredeus) [inline package.json:4]: Can this file be reverted to what we have in develop?

File: pages/[id]/edit.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import { useRouter } from 'next/router'
```
Comment (amaredeus) [inline pages/[id]/edit.js:1]: packages/react-app/src

File: pages/[id]/index.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
```
Comment (amaredeus) [inline pages/[id]/index.js:1]: packages/react-app/src

File: pages/_app.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import '../css/style.css'
```
Comment (amaredeus) [inline pages/_app.js:1]: packages/react-app/src

File: pages/api/bounties/[id].js [hunk 10]
```diff
@@ -0,0 +1,56 @@
+import dbConnect from '../../../utils/dbConnect'
```
Comment (amaredeus) [inline pages/api/bounties/[id].js:1]: packages/react-app/src

File: pages/api/bounties/index.js [hunk 11]
```diff
@@ -0,0 +1,32 @@
+import dbConnect from '../../../utils/dbConnect'
```
Comment (amaredeus) [inline pages/api/bounties/index.js:1]: packages/react-app/src

File: pages/index.js [hunk 12]
```diff
@@ -0,0 +1,61 @@
+import Link from 'next/link'
```
Comment (amaredeus) [inline pages/index.js:1]: packages/react-app/src

File: pages/new.js [hunk 13]
```diff
@@ -0,0 +1,18 @@
+import Form from '../components/Form'
```
Comment (amaredeus) [inline pages/new.js:1]: packages/react-app/src

File: public/zeit.svg [hunk 14]
```diff
@@ -0,0 +1,10 @@
+<svg width="82" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (amaredeus) [inline public/zeit.svg:1]: packages/react-app/src

File: utils/dbConnect.js [hunk 15]
```diff
@@ -0,0 +1,45 @@
+import mongoose from 'mongoose'
```
Comment (amaredeus) [inline utils/dbConnect.js:1]: packages/react-app/src

**Review by amaredeus — CHANGES_REQUESTED** [review]
Pretty much the source code files should be placed in then we react-app directory and that should trigger the yarn build correctly

**Comment by Oktalize** [comment]
@Behold3th per our discussion is it okay if we close this PR. We can work on my new branch instead now that i've merged in your changes with it?

**Comment by Behold3th** [comment]
> @Behold3th per our discussion is it okay if we close this PR. We can work on my new branch instead now that i've merged in your changes with it?

yep, all good!

**Pull request closed without merging**